kuma/static/js/interactive.js

```diff
@@ -7,6 +7,84 @@
         window.mdn.interactiveEditor.editorUrl ||
         'https://interactive-examples.mdn.mozilla.net';
 
+    /**
+     * Returns the interactive examples iframe performance entry
+     * @param {Array} perfEntries - The array of performance entries
+     */
+    function getInteractiveExamplesPerfEntry(perfEntries) {
+        for (var i = 0, l = perfEntries.length; i < l; i++) {
+            var currentEntry = perfEntries[i];
+            if (
+                currentEntry.initiatorType &&
+                currentEntry.initiatorType === 'iframe' &&
+                currentEntry.name.indexOf(
+                    'interactive-examples.mdn.mozilla.net'
+                ) > -1
+            ) {
+                return perfEntries[i];
+            }
+        }
+    }
+
+    /* Ensure there is an iframe present and that performance is
+       supported before gathering performance metric */
+    if (iframe && performance !== undefined) {
+        document.addEventListener('readystatechange', function(event) {
+            if (event.target.readyState === 'complete') {
+                var interactiveExamplesPerfEntry = {};
+                var perfEntries = performance.getEntriesByType('resource');
+                var mainFetchStart = 0;
+                var iframeFetchStart = 0;
+                var iframeFetchStartSinceUnixEpoch = 0;
+
+                if (perfEntries === undefined || perfEntries.length <= 0) {
+                    console.info('No performance entries was returned');
+                    return;
+                }
+
+                interactiveExamplesPerfEntry = getInteractiveExamplesPerfEntry(
+                    perfEntries
+                );
+
+                mainFetchStart = performance.timing.fetchStart;
+                iframeFetchStart = Math.round(
+                    interactiveExamplesPerfEntry.fetchStart
+                );
+                iframeFetchStartSinceUnixEpoch =
+                    mainFetchStart + iframeFetchStart;
+
+                // var iframeLoadTime = interactiveExamplesPerfEntry.responseEnd - interactiveExamplesPerfEntry.startTime;
+                // ga('send', {
+                //     'eventCategory' : 'Interactive Examples',
+                //     'eventAction'   : 'iFrame load time',
+                //     'eventLabel'    : 'Performance Events',
+                //     'eventValue'    : (iframeLoadTime / 1000),
+                //     'hitType': 'timing'
+                // });
+                console.log(
+                    (interactiveExamplesPerfEntry.responseEnd -
```

Comment:
Can we collect the metric in milliseconds for additional precision? Maybe with `Math.round()` though to clean it up slightly.